test/e2e/aadpodidentity_test.go

```diff
@@ -708,42 +708,94 @@ var _ = Describe("Kubernetes cluster using aad-pod-identity", func() {
 	})
 
 	It("should pass validation by bypassing nmi using azurepodidentityexception crd", func() {
+		// Creates 2 pods with labels defined in AzurePodIdentityException
+		// Creates 1 pod that needs to go through pod-identity
+		// Validates the mixed scenario works as expected.
+
 		// Assign user assigned identity to every node
 		nodeList, err := node.GetAll()
 		Expect(err).NotTo(HaveOccurred())
 
-		enableUserAssignedIdentityOnCluster(nodeList, keyvaultIdentity)
+		enableUserAssignedIdentityOnCluster(nodeList, fmt.Sprintf("%s-%d", keyvaultIdentity, 1))
+		enableUserAssignedIdentityOnCluster(nodeList, fmt.Sprintf("%s-%d", keyvaultIdentity, 2))
+
+		enableSystemAssignedIdentityOnCluster(nodeList)
```

Comment:
Not sure why we are enabling system assigned identity in this test ?

Reply:
How do we get this to work with a setup with system assigned identity cluster (assuming that #319 is through) ?